documentation/modules.md

```diff
@@ -1,11 +1,16 @@
-# Modules
+# Modules overview
 
 Many of the modules on the Puppet Forge include helpful plans and tasks that you
 can install with Bolt and use in your workflows. Modules also allow you to share
-your own custom Bolt content, such as tasks and plans, on the Forge or use your
-Bolt content programmatically.
+your own custom Bolt tasks and plans on the Forge or use your Bolt content
+programmatically.
 
-If you're looking for a way to use or create Bolt content that is specific to
-your organization's infrastructure, and share that content with your team, a
-Bolt project may be all you need. For more information, see [Bolt
-projects](projects.md).
\ No newline at end of file
+If you're interested in creating Bolt orchestration that is
+specific to your organization's infrastructure, and sharing that content with
+your team, you can build a Bolt project that includes modules from the Forge.
+
+## 🧪 Managing modules in Bolt projects
```

Comment:
Should this still link to the other modules pages below? It looks like Dita didn't include the whole nav tree: https://puppet-docs-preview.netlify.app/docs/bolt/latest/modules.html

Reply:
Ah, I forgot to change the ditamap back to the way it was before. I was originally planning on fleshing this page out more, but I ended up deciding it was more or less fine the way it was.